### DISCUSSION TITLE
ServiceMesh/overview

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#248] [General]
# ServiceMesh/overview

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/ServiceMesh/overview.html

**Comment by useforgithub** [comment]
作者好，SSL证书该换了，浏览器一直弹错误，烦

**Comment by zjyyzhengli** [comment]
Reference 中的第二条后面的文字重复了（**以便任何人都可以复核结果**）
